libledger/Ledger.cpp

```diff
@@ -120,8 +120,17 @@ void Ledger::initIniConfig(std::string const& iniConfigFileName)
                      << LOG_DESC("initTxPoolConfig/initSyncConfig/initTxExecuteConfig")
                      << LOG_KV("configFile", iniConfigFileName);
     ptree pt;
-    /// read the configuration file for a specified group
-    read_ini(iniConfigFileName, pt);
+    try
+    {
+        /// read the configuration file for a specified group
+        read_ini(iniConfigFileName, pt);
+    }
+    catch (const std::exception& e)

```

Comment:
If this is wrong, fisco-bcos should exit. if not catch, it will exit. this catch make fisco-bcos can't exit?

Reply:
done. pls refer to https://github.com/FISCO-BCOS/FISCO-BCOS/pull/708/files#diff-708d2961ab288bc22804247af5e4c497R123